src/index.ts

```diff
@@ -39,6 +39,7 @@ export interface CoreOptions {
   multipart?: RequestPart[];
   forever?: boolean;
   pool?: HttpsAgentOptions | HttpAgentOptions;
+  followRedirect?: boolean;
```

Comment:
by default `node-fetch` limits to 20 redirections, while `request` is 10. should this also support overriding that default?
```
maxRedirects?: number;
```

`request`'s `maxRedirects`: https://github.com/request/request/blob/536f0e76b249e4545c3ba2ac75e643146ebf3824/lib/redirect.js#L13
`node-fetch`'s `follow`: https://github.com/node-fetch/node-fetch/blob/8a3d2638ae491bed88861643f1ff8bbc0848b5cc/src/request.js#L132-L134